aiohttp/http_parser.py

```diff
@@ -698,8 +698,26 @@ def __init__(self, out: StreamReader, encoding: Optional[str]) -> None:
             if not HAS_BROTLI:  # pragma: no cover
                 raise ContentEncodingError(
                     'Can not decode content-encoding: brotli (br). '
-                    'Please install `brotlipy`')
-            self.decompressor = brotli.Decompressor()
+                    'Please install `Brotli`')
+
+            class BrotliDecoder(object):
```

Comment:
In Python 3 all classes are new-style already:
```suggestion
            class BrotliDecoder:
```

Reply:
Updated, thanks!